content/en/docs/concepts/storage/volumes.md

```diff
@@ -1034,6 +1034,11 @@ As a result, operators do not have to make any
 configuration changes to existing Storage Classes, PersistentVolumes or PersistentVolumeClaims
 (referring to in-tree plugins) when transitioning to a CSI driver that supersedes an in-tree plugin.
 
+{{< note >}}
+* The existing PV created by in-tree volume plugin can still be used in the future without any configuration changes even after the `CSIMigration` is completed. 
```

Comment:
- "The" looks unnecessary: `The existing PV` -> `Existing PVs`
- I think it is better to add `and a corresponding in-tree volume plugin is removed` after `after the CSIMigration is completed` for further clarification. While I know the scope of CSI migration effort includes removals of corresponding in-tree volume plugins, I still think this kind of clarification helps operators who run kubernetes.

```suggestion
* Existing PVs created by in-tree volume plugin can still be used in the future without any configuration changes even after the `CSIMigration` is completed and corresponing in-tree volume plugins are deleted.
```

Reply:
Good point, I'm also agree this suggestion.